core/src/main/java/hudson/model/Computer.java

```diff
@@ -703,6 +704,9 @@ public void setTemporarilyOffline(boolean temporarilyOffline) {
      *      offline.
      */
     public void setTemporarilyOffline(boolean temporarilyOffline, OfflineCause cause) {
+        if (temporarilyOffline && cause == null) {
+            LOGGER.log(Level.WARNING, "Parameter mismatch in setTemporarilyOffline. You want set node temporarily-offline, but the cause is null, {0}", new Throwable().getStackTrace());
```

Comment:
As above.

Reply:
I know. The same as above. But in this case we shall really fix this. It might brings some side effects now, and you will newer see, why is the not offline (same on my site when I try to set node offline by groovy script ;-) ) I will remove the message and set the variable *temporarilyOffline* to false in this case.